### PR TITLE
Pranita_Assignment_8_Done

### PR DESCRIPTION
**Pull request by PranitaPJun20**
In this assignment added Print Current balance logic.Used balance as a static variable.
Added 5 methods as per requirement and shown two users transaction summary.

### CODE REVIEW

File: src/pranitaPJun20/BankingSystem.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+public class BankingSystem {
+	//Instance Variables
+	int debitCount=0;
+	int creditCount=0;
+	int printBalanceCount=0;
+	//Static variables
+	static int balance=100000;
+	static int staticDebitCount=0;
+	static int staticCreditCount=0;
+	static int staticPrintBalCount=0;
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/BankingSystem.java]: Remove initialization of all instance variables. And fro initial balance you can call credit() method first.

File: src/pranitaPJun20/BankingSystem.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	
+		//Method to Debit amount
+		void debitAmnt(int amountDebited ){
+			debitCount= debitCount+1;
+			//debitCount++ ; //we can use this also
+			//debitCount+=1 ;//we can use this command also	
+			staticDebitCount=staticDebitCount+1;
+			balance= balance-amountDebited;
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/BankingSystem.java:20]: You should write the condition to verify the insufficient balance.

File: src/pranitaPJun20/BankingSystem.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+		}
+		//Method to credit amount
+		void creditAmnt(int amountCredited){
+			creditCount= creditCount+1;
+			staticCreditCount=staticCreditCount+1;
+			balance= balance-amountCredited;
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/BankingSystem.java]: Do copy-paste code. Here it should be addition operation.

File: src/pranitaPJun20/BankingSystem.java (hunk 2 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/BankingSystem.java:20]: Done.Added condition to handle insufficient balance in account

File: src/pranitaPJun20/BankingSystem.java (hunk 1 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/BankingSystem.java]: Done

File: src/pranitaPJun20/BankingSystem.java (hunk 2 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/BankingSystem.java:20]: Done

File: src/pranitaPJun20/BankingSystem.java (hunk 3 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/BankingSystem.java]: Yes.Changed

File: src/pranitaPJun20/BankingSystem.java [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+	static int staticPrintBalCount;
+	
+//Method to Debit amount
+		void debitAmnt(int amountDebited ){
+			debitCount= debitCount+1;
+			//debitCount++ ; //we can use this also
+			//debitCount+=1 ;//we can use this command also	
+			staticDebitCount=staticDebitCount+1;
+			balance= balance-amountDebited;
+				if(balance<=100) {
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/BankingSystem.java:21]: This is not the condition that we expected. Please review https://github.com/KrishnaKTechnocredits/JAVATechnoJun20/pull/100

**Pull request merged**